src/software/ai/hl/stp/play_info.h

```diff
@@ -3,22 +3,68 @@
 #include <string>
 #include <vector>
 
-struct PlayInfo
+/**
+ * A description of the play.  Contains the type of play being
+ * run, its name, and the tactic assignments for all the robots
+ * on the field.
+ */
+class PlayInfo
 {
+   public:
+    /**
+     * Constructs a PlayInfo object.
+     */
+    explicit PlayInfo();
```

Comment:
Can we add a constructor that also takes arguments for the class members? That way we aren't forced to use the class like a struct and set the members one-by-one.